src/protocol-next/xdr/Stellar-transaction.x

```diff
@@ -32,6 +32,13 @@ struct DecoratedSignature
     Signature signature; // actual signature
 };
 
+// Ledger key sets touched by a smart contract transaction.
+struct Footprint
```

Comment:
maybe use a qualified name like "LedgerFootprint" when I hear "footprint" I think "certificate footprint" (sorry, this should be a CAP comment)

Reply:
Ok, changed to `LedgerFootprint`